private-api.go

```diff
@@ -39,6 +39,51 @@ type Balances struct {
 	Pair map[string]float64
 }
 
+func (c *Client) GetCompleteBalances(ctx context.Context) (CompleteBalances, error) {
```

Comment:
__[golint]__
* [golint] exported method Client.GetCompleteBalances should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->